File: Player.cpp
//This is the Player. 
//This is your avatar. 

#include "Player.hpp"
#include <iostream>
using namespace std;

void setName(){
    cout << "What is your name?\n";
    cin >> name;
}

double setMoney(){
    cout << "How much money do you have?\n$";
    cin >> money;
}

File: Player.java
public class Player {
    
    private String name;
    private double money = 100.00;
    
    public void setName(String n){
        name = n;
    }
    public void setMoney(double m){
        money = m;
    }
    public String getName() {
        return name;
    }
    public double getMoney(){
        return money;
    }
    public void summary(){
        System.out.println("My name is " + getName() +"\nI have " + getMoney() + " dollars currently");
    }
}